deployment/ansible/roles/nyc-trees.beaver/templates/django.conf.j2

```diff
@@ -0,0 +1,5 @@
+[/var/log/{upstart/nyc-trees-app.log,nyc-trees-app.log}]
```

Comment:
For my education, is the curly brace section indicating that we want to pull from both files?